src/browser/reset.js

```diff
@@ -8,7 +8,12 @@ const DEFAULT_LANG = 'en',
 let currentLang = undefined,
 	currentRtl = false,
 	currentViewportHeight = 0,
-	currentViewportWidth = 0;
+	currentViewportWidth = 0,
+	shouldResetMouse = true;
```

Comment:
This will cause the mouse to be reset for the first test in every test file. If we think we can count on Playwright to start the mouse at (0, 0) we could default this to `false`, saving maybe an extra 5 seconds overall.

Reply:
Looking at the code, I think we can assume (0, 0) starting position:
https://github.com/microsoft/playwright/blob/90c92c5724a25e1d9717908581abd35cc0410bde/packages/playwright-core/src/server/input.ts#L165

Reply:
Sounds good, it's also really easy to switch if we learn otherwise